finance/proposal-calls/Moore_2021.md

```diff
@@ -0,0 +1,92 @@
+# Call for Astropy ideas / projects to be funded in 2021
+
+## Goals
+Projects and ideas funded through this process shall improve The Astropy Project. This includes code for the core repository, for coordinated or affiliated packages, infrastructure (e.g. testing, CI), tutorials, workshops, notebooks, community engagement, inclusion, diversity and empowerment, travel to conferences or meetings, talks, mentoring, and anything else broadly Project related. This does not have to be a new initiative, but can be ongoing work (e.g. maintain the CI), but cannot include work that is already otherwise paid for (e.g. if you have a NASA grant to maintain the CI, you cannot bill for the same work twice).
+
+Suggested projects shall foster the Astropy Project goals of an open and inclusive community, shared work, and cooperation and shall be proposed and executed in this manner.
+
+Total funding for this call is coming from the Moore grant and 200,000 $ are available in total in the period spring 2021 to spring 2022.
+
+Project suggetions need to clearly identify what the funding is used for (e.g. "pay YYY $ to add Klingon Standard Time to astropy.time" or "organize astropy workshop to engage amateur astronomer community in Antarctica")
+
+*Note: You can think of this as an internal "proposal process" to allow the community to select exactly where the money that the Astropy Project has will be spend. To avoid confusion with the external proposal that gave the money to Astropy (here: Moore foundation), we use the term "suggested project" and "project team" here.*
+
+## Details
+
+- Source of money: Moore grant
+- Specific area: Most of the money can be distributed to any area deemed important by the community. However, we do have specific budgets for work related to spectroscopy (22500 US $), mentoring contributors (15000 US $), and first contributor workshops (12000 US $).
+- Restrictions: This is a world-wide call, open to all past, current, and future contributors to the Astropy project. (The only restriction is that money cannot be transferred to countries that the United States does not allow banking transactions with, e.g. North Korea).
```

Comment:
Give a link to an official document that lists the countries in that list, or list all of them here as it's not very inclusive to expect everyone to know who is restricted and frankly it's also a moving goalpost with a risk to hearsay (as countries are added and removed from that list all the time).

Reply:
Exactly. If I had an exact list that I knew to be valid in the future, I would work at Wall Street and not here ;-)
We can't transfer money when it is illegal to do so - but unfortunately, nothing stops to make US from changing their laws after I merge this PR. Yes, it's not inclusive and I don't like it, but there is nothing we can do about it. I know about certain restrictions form the newspaper and don't have any more information about it that I could give. If there is an actual need to find out more, I'm willing to to through the US government websites to find out more but I don't have time to do that unless there is an actual need to do so. Sorry.

Reply:
I appreciate the suggestion, but I don't think I (or the interim finance committee) are the right people to so that. My employer has a whole office of legal staff and they often don't know for things like this.

Reply:
here's a link, but I'm not sure a non-lawyer would be able to interpret the information there very easily: https://home.treasury.gov/policy-issues/financial-sanctions/sanctions-programs-and-country-information

Reply:
Yes, having that link is very helpful I think, as it at least provides a starting point to see the list might be much wider than the obvious, e.g. NK, Iran, Cuba, in fact a few of those that have recent updates are ones where we had GSoC participation from in the past and/or has reasonably sizeable astro community. 

Also, I think if the link doesn't make it to the doc, then rephrasing to something more concrete is helpful, googling to banking restrictions is pretty difficult. E.g. GSoC phrases it (https://summerofcode.withgoogle.com/rules/):
"not be based in a United States embargoed country, or otherwise prohibited by applicable export controls and sanctions programs; and", outreach also includes citizens and nationals of such countries in addition to residents, but both talks about export controlled countries, as the whole issue is not really about the banking per se.